universe/config.json

```diff
@@ -29,6 +29,21 @@
             "description":"Broker rollout strategy. See documentation. [INSTALL, STAGE]",
             "type":"string",
             "default":"INSTALL"
+          },
+          "enable_replacement":{
+            "description":"Enable automated replacement of Brokers. WARNING: May cause data loss. See documentation.",
+            "type":"boolean",
+            "default":false
+          },
+          "recover_in_place_grace_period_mins":{
```

Comment:
any reason for one to be in units of minutes and the other to be in units of seconds?